targets/targets.json

```diff
@@ -7428,6 +7428,12 @@
             "IAR",
             "ARMC6"
         ],
+        "components": [
+            "FPGA_CI_TEST_SHIELD"
```

Comment:
Please remove `FPGA_CI_TEST_SHIELD` as shouldn't be added by default

Reply:
OK, we will change it.